[2025-05-05T14:11:14.220+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-05T14:11:14.236+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-05-05T14:11:12.723936+00:00 [queued]>
[2025-05-05T14:11:14.245+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-05-05T14:11:12.723936+00:00 [queued]>
[2025-05-05T14:11:14.246+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-05-05T14:11:14.259+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_raw_data> on 2025-05-05 14:11:12.723936+00:00
[2025-05-05T14:11:14.264+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=997) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-05T14:11:14.266+0000] {standard_task_runner.py:72} INFO - Started process 1028 to run task
[2025-05-05T14:11:14.266+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'extract_raw_data', 'manual__2025-05-05T14:11:12.723936+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/lead_data_pipeline.py', '--cfg-path', '/tmp/tmpqm88i7g1']
[2025-05-05T14:11:14.267+0000] {standard_task_runner.py:105} INFO - Job 19: Subtask extract_raw_data
[2025-05-05T14:11:14.311+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-05-05T14:11:12.723936+00:00 [running]> on host 433e63c87a30
[2025-05-05T14:11:14.393+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-05T14:11:12.723936+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-05T14:11:12.723936+00:00'
[2025-05-05T14:11:14.394+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-05T14:11:14.409+0000] {lead_data_pipeline.py:189} INFO - Started batch processing for 2025-05-05.
[2025-05-05T14:11:14.410+0000] {lead_data_pipeline.py:33} INFO - Created synthetic data for RO country code.
[2025-05-05T14:11:41.804+0000] {lead_data_pipeline.py:100} INFO - Written 100372 records to the CSV file.
[2025-05-05T14:11:42.337+0000] {lead_data_pipeline.py:158} INFO - Added/Updated UUIDs in the dataset.
[2025-05-05T14:11:42.338+0000] {lead_data_pipeline.py:197} INFO - Finished batch processing 2025-05-05.
[2025-05-05T14:11:42.338+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-05T14:11:42.349+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-05T14:11:42.350+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=extract_raw_data, run_id=manual__2025-05-05T14:11:12.723936+00:00, execution_date=20250505T141112, start_date=20250505T141114, end_date=20250505T141142
[2025-05-05T14:11:42.372+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-05T14:11:42.395+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-05T14:11:42.397+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
